lib/browser/crypto.ts

```diff
@@ -12,25 +12,34 @@
  */
 
 import * as Crypto from "crypto-js";
-import { fromUtf8 } from "@aws-sdk/util-utf8-browser";
 import { Hashable } from "../common/crypto";
 
 export { Hashable } from "../common/crypto";
 
 /**
- * Object that allows for continuous MD5 hashing of data.
- *
- * @category Crypto
+ * CryptoJS does not provide easy access to underlying bytes.
+ * As a workaround just dump it to a string and then reinterpret chars as individual bytes.
+ * TODO: long term we would probably want to move to WebCrypto for SHA's and some other 3p for crc's and md5.
+ * @param hash 
+ * @returns 
  */
-export class Md5Hash {
-    private hash?: Crypto.WordArray;
+function hashToUint8Array(hash: Crypto.WordArray) {
+    return Uint8Array.from(hash.toString(Crypto.enc.Latin1).split('').map(c => c.charCodeAt(0)));;
```

Comment:
why do we use `Crypto.enc.Latin1`?

https://github.com/brix/crypto-js/issues/274
Looks like there were some discussions about this.

Reply:
Its mostly for simplicity purposes. Dumping to hex and then parsing from it is slightly more code. 
I tried out parsing the array out of the words initally and it was not that much of perf difference and would require updating types dependency, which breaks mqtt code. Instead of mucking further with cryptoJS and its deps at that point i think it would be better to spend that time on dropping cryptoJS altogether, so i just went with whatever works with current deps.

Reply:
Maybe renaming to `wordArrayToUint8Array`?

Maybe add some comments about how `Crypto.enc.Latin1` and `charCodeAt` to convert the buffer back and forth? 

So, firstly, I thought we should use base64 as we commonly used. But, after digging, it seems not a directly match when converted to string. And the [documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/charCodeAt) of `charCodeAt` said it used `utf-16`?

Agreed on whatever works, but I feel those encoding are really confusing.

Reply:
completely agree and i think eventually it makes sense to move off CryptoJS and use WebCrypto as well. 
current complication is that webcrypto only has shas and we need alternatives for crc's and md5 (if we want to keep a browser version).
I added a comment, but the jist of it is that latin1 is 8bit encoding (my understanding its the same as 8bit ascii variant) and charCodeAt just retrieves the corresponding byte value (it can return values bigger than a byte for unicode, but in this case we are limited by chars not being unicode to byte size values)